content/docs/developing-operators/getting-started.md

```diff
@@ -93,16 +91,16 @@ At the same time, `deploy` plan is the most important plan within your operator
 
 Plans allow operators to see what the operator is currently doing, and to visualize the broader operation such as for a config rollout. The fixed structure of that information meanwhile makes it straightforward to build UIs and tooling on top.
 
-## Params File
+# Params File
 
 `params.yaml` defines all parameters that can be used to customize your operator installation.
 They are described in more detail in the [document about operator parameters](operator-parameters.html).
 
-## Templates
+# Templates
 
```

Comment:
This should probably link to `templates.md` for details about Go Templates, Sprig, etc.

Reply:
Yes, will go through this document in a follow-up PR.